framework/web/UrlManager.php

```diff
@@ -242,6 +242,12 @@ public function parseRequest($request)
 
             Yii::trace('No matching URL rules. Using default URL parsing logic.', __METHOD__);
 
+            // before processing pathInfo and suffix there should not be end multiple slashes in $pathInfo
+            $endSlashPattern = '//';
```

Comment:
Why do we need a separate variable for this? Comment above is enough to describe, what's going here